pipelines/templates/run-integrationtests.yml

```diff
@@ -32,6 +54,12 @@ steps:
     winget source add -n "winget-cli-restsource" -a $(RestSourceUrl) -t "Microsoft.Rest"
   displayName: "Setup test pre-requisites"
 
+- powershell: |
+    & winget source add -n "winget-cli-restsource" -a "$(RestSourceUrl)" -t "Microsoft.Rest"
+    Write-Host "Added rest source to winget cli."
+
+  displayName: "Rest source"
```

Comment:
As this is the rest source, not the client source, I don't think we should be using the client to validate it's correctness. We want to make sure that it works using standard rest tooling independent from the client. The client source code should be responsible for making sure it is interacting with REST in an appropriate manor.

Reply:
Removed